geometry_msgs/Point point1
geometry_msgs/Point point2
geometry_msgs/Point point3
---
geometry_msgs/Pose alignedpose
float32 page_width
float32 page_height
